blocked-edges/4.3.1.yaml

```diff
@@ -0,0 +1,4 @@
+to: 4.3.1
+from: .*
```

Comment:
why are we blocking 4.3.0 to 4.3.1 ?

Reply:
I guess I was just copying #100's [broad block][1] back.  Now that we're down to just the TargetDown/container-port bug, I've pushed ce7fc11 -> 5252c62 to restrict the blocks to 4.2 -> 4.3.

[1]: https://github.com/openshift/cincinnati-graph-data/pull/100/files#diff-bd1c08aa70b14c3a8b60033684b0af71R3

Reply:
![graph](https://user-images.githubusercontent.com/3125299/77163719-ae69c600-6ad4-11ea-98b4-2e20d14e4c1f.JPEG)

FYI, The current graph has only edge from 4.3.0 to 4.3.1. If we are enabling 4.3.* to 4.3.2, we need to go back check the telemetry dashboard, even release CI jobs etc as I do not believe these things are checked at all in the past

Reply:
I would recommend this PR just replicates the graph what we have already  and if we want to change the graph then we send a new PR where we do the normal checks around the health of the edge.